### ISSUE TITLE
check EU baseline distribution of IAS of Union concern in EASIN

### ISSUE THREAD

**Issue by timadriaens** [inbo/alien-species-checklist#84]
this issue was moved to the appropriate [repo](https://github.com/inbo/invasive-t0-occurrences/issues/86)

**Comment by timadriaens** [comment]
- [ ] remove Spiraea tomentosa L. from subset and check/update original T0 file EU concern status

- [ ] remove Lepomis gibbosus (Linnaeus, 1758) and check/update original T0 file EU concern status

- [ ]  remove Lupinus polyphyllus Lindl.  and check/update original T0 file EU concern status 

- [ ] id. Myriophyllum heterophyllum Michx.       

- [ ] id. Nyctereutes procyonoides (Gray, 1834)          

- [ ] id. Ondatra zibethicus (Linnaeus, 1766)  
     

- [ ] check if all species occurring in Belgium are there: the following are lacking

![image](https://cloud.githubusercontent.com/assets/16116743/21982062/d7ae10ec-dbea-11e6-81fc-f3a9392c3647.png)

**Comment by timadriaens** [comment]
we probably have to move this issue to T0

**Comment by timadriaens** [comment]
Non-validated records were discarded from the analysis. This was done based on IdentificationVerificationStatus. We kept the following values:

N/A (institution code INBO) (66026)
Goedgekeurd op basis van bewijsmateriaal (8211 records)
Goedgekeurd op basis van expertoordeel (1558)
Goedgekeurd op basis van kennisregels (4832)
j (= validated data from mammal working group) (407)
validÃ© avec document (608)
validÃ© sans document  (683)
validÃ© sur base de rÃ¨gles (4088)

Records without value IdentificationVerificationStatus were mostly records from INBO. These are considered validated. The other non-INBO records need to be checked.

As for the non-validated records, we need to do a check of the verification status per species as for some conspicuous species we might easily assume even non-validated records are ok. This is done in a separate R script for preliminary data analysis.

**Comment by timadriaens** [comment]
date cut-off: preliminary analysis of number of records per year and per species + number of sites per species per year; Decide on cut-off based on this.

**Comment by timadriaens** [comment]
some further information from EASIN on how to deal with eradicated species and which cut-off dates to consider:

Dear colleagues,

Thank you for your e-mail.
Regarding your questions:

1. Country level:
Concerning time period: the baseline should reflect the current status and situation of the Union concern IAS within each MS. No specific time period has been set in terms of years, but it is up to MS to decide which data reflect best today's situation. Most frequently, "historical" records dating before 1960s and never reported again are not taken into consideration for the baseline, but this is just indicative.
Concerning casual records: there are 3 different status categories per species in the baseline: established, casual and absent. Casual are those corresponding to rare, occasional records, and/or species that are not reproducing and/or species not overwintering; in overall casual records correspond to non-established populations, that cannot survive by their own. Casual species should be marked as suggested in the baseline protocol.
Concerning eradicated records: these records should not be considered for the baseline, since the species is no more present within the country; eradicated = absent.

2. Grid level:
Concerning time period: as above.
Concerning casual records: these should be marked also for the grid level. You can easily do this by adding an extra column in the GIS Attributes Table, naming it "Status" and fill-in by "C = casual" or "E = established".
Concerning eradicated records: as in the country level.

Please let us know if you have any other question.

We appreciate your collaboration


The EASIN team

European Commission
DG Joint Research Centre
Directorate D – Sustainable Resources
Unit D.02 Water and Marine Resources

Via Enrico Fermi, 2749 – TP 270
I-21027 Ispra (VA) - Italy
http://easin.jrc.ec.europa.eu




-----Original Message-----
From: BRANQUART Etienne [mailto:etienne.branquart@spw.wallonie.be]
Sent: Thursday, December 01, 2016 4:35 PM
To: Konstantinos Tsiamis <konstantinos.tsiamis@jrc.ec.europa.eu>
Cc: maurits.vandegehuchte@lne.vlaanderen.be; bram.dhondt@lne.vlaanderen.be; 'ADRIAENS, Tim' <tim.adriaens@inbo.be>; PREVOT Céline <celine.prevot@spw.wallonie.be>; easin@jrc.ec.europa.eu; env-IAS@ec.europa.eu
Subject: RE: Establishment of baseline distribution of 37 species on Union list: Belgium country and grid level

Dear colleagues,

I'm working hardly with my Belgian colleagues to gather distribution data and check the maps and shapefiles you provided.

In order to be sure to fit your expectations, I would like to ask you some questions related tot he baseline information asked by you we didn't find in the guidance document:

1. Country level information

The information illustrated on your website is binary, i.e. species present or absent. Could you please specify:
- what is the time period to be considered for species presence (starting and ending date)?
- do we have to consider a species with casual observations as present in the country? Is it a threshold concerning the number of casual data to this purpose? Example : if a bird has been observed only once a long time ago, do we have to consider it as present?
- how should we deal with species that have been eradicated like the Pallas squirrel? Is is not present anymore. Don't we have to consider "eradicated" on top of absent or present?

Our suggestion would be to consider 4 different status categories per species, i.e. absent from the country, casual, established and eradicated.

2. Grid level information

As for the country level information, we would like to know:
- what is the time period to be considered for the exercise (starting and ending date)?
- how to deal with casual observations?
- how to deal with eradicated populations?

Many thanks in advance for your answer.

Best regards,

Etienne Branquart

-----Message d'origine-----
De : Konstantinos Tsiamis [mailto:konstantinos.tsiamis@jrc.ec.europa.eu]
Envoyé : jeudi 14 juillet 2016 15:58
À : BRANQUART Etienne
Cc : maurits.vandegehuchte@lne.vlaanderen.be; bram.dhondt@lne.vlaanderen.be; 'Sonia Vanderhoeven'; 'ADRIAENS, Tim'; PREVOT Céline; CAIGNET Isabelle; easin@jrc.ec.europa.eu; env-IAS@ec.europa.eu Objet : RE: Establishment of baseline distribution of 37 species on Union list: Belgium country and grid level

Dear colleagues,

Thank you for your e-mail.

As you may know, EASIN maps rely on spatial data harvested from the EASIN data partners (such as GBIF, but also others:
http://easin.jrc.ec.europa.eu/Partners/Partners). Therefore, in case of unpublished data of spatial distribution existing, EASIN cannot provide accurate information.
As a result, there could be indeed discrepancies between EASIN and your spatial data.
It is crucial that an accurate and updated baseline of the 37 Union concern IAS can be achieved. So, please take the all time needed for the baseline check, even beyond 30 September.
In addition, please have in mind that you could provide directly to EASIN your spatial data shape files, and you just indicate through the EASIN shape file which EASIN records are those that you do not agree with (by also providing justification). That could be a more time-efficient way to complete the baseline validation.

Please do not hesitate to contact us in case of any question or issue.

Best regards
The EASIN team







-----Original Message-----
From: BRANQUART Etienne [mailto:etienne.branquart@spw.wallonie.be]
Sent: Wednesday, July 13, 2016 10:06 PM
To: easin@jrc.ec.europa.eu; env-IAS@ec.europa.eu
Cc: maurits.vandegehuchte@lne.vlaanderen.be; bram.dhondt@lne.vlaanderen.be; Sonia Vanderhoeven <sonia.vanderhoeven@gmail.com>; ADRIAENS, Tim <tim.adriaens@inbo.be>; PREVOT Céline <celine.prevot@spw.wallonie.be>; CAIGNET Isabelle <isabelle.caignet@spw.wallonie.be>
Subject: RE:Establishment of baseline distribution of 37 species on Union
list: Belgium country and grid level

Dear EASIN and IAS teams,

Thank you very much for your message, the great work you did and the files you shared with us.

Together with Belgian colleagues in cc, I made a quick comparison of the EASIN distribution layer with our own reference layer for the 37 species. We found huge discrepancies between both sources, with quite a lot of data missing in EASIN (especially data not yet published on GBIF) but also some occurrences in EASIN not found in our database.

It means that quite a lot of work is needed to review, validate and complete the distribution data in EASIN. We realised that this exercise will be already difficult for Belgium, but probably even more for federal states where distribution data are spread over many different entities as it is the case in Germany. We are afraid it will not be possible to provide a feedback by September 30th and we would like to ask you to stick with the original deadline that has been proposed during the last IAS Committee meeting, i.e.
the end of this year (= December 31st 2016).

We sincerely hope that you could agree with this suggestion and postpone the deadline taking into account that errors in this baseline distribution layer should be minimised.

Very best regards,

Etienne Branquart, for the Belgian IAS team

________________________________________
De : EASIN [easin@jrc.ec.europa.eu]
Envoyé : mardi 12 juillet 2016 18:22
À : CAIGNET Isabelle; maurits.vandegehuchte@lne.vlaanderen.be
Cc : env; eugenio; konstantinos
Objet : Establishment of baseline distribution of 37 species on Union list:
Belgium country and grid level

Dear colleagues,

As discussed in the 6th IAS Committee on 22 June, Member States are invited to liaise with the EASIN team of JRC for verifying the "baseline distribution data" in EASIN for the 37 species on the first Union list.

To this end, we kindly request your authorities to review the current information in EASIN on the distribution in your country of the 37 species on the first list of IAS of Union concern and provide feedback on this by 30 September. The purpose is to establish the baseline distribution of these species in both country and grid 10 x 10 km level.
Detailed instructions on how to carry out this can be found in the attached pdf document (EASIN baseline protocol). Shape files for the grid 10 x 10 km and related references (excel file) are also attached.

For any related questions, please contact us by e-mail at easin@jrc.ec.europa.eu<mailto:easin@jrc.ec.europa.eu>

Sincerely,
The EASIN team

**Comment by peterdesmet** [comment]
Closing here.
